tools/probes/probes_main.c

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2018 Intel Corporation. All rights reserved.
+//
+// Author: Adrian Bonislawski <adrian.bonislawski@intel.com>
+
```

Comment:
Some description in comments on what this tool does.

Reply:
Done